tests/operator/operator.go

```diff
@@ -2984,6 +2991,7 @@ spec:
 		})
 
 		AfterEach(func() {
+			// Reset the feature gate to its default value after running any of these tests.
```

Comment:
One question shouldn't this restore the feature gates. What if the feature gate was enabled?

Reply:
You are right! Updated the PR.